react/components/SearchResult.js

```diff
@@ -15,14 +15,14 @@ class SearchResult extends Component {
   static propTypes = searchResultPropTypes
 
   static defaultProps = {
-    hiddenFacets: {
-      layoutMode1: LAYOUT_MODE[0].value,
-      layoutMode2: LAYOUT_MODE[1].value,
+    mobileLayout: {
+      mode1: LAYOUT_MODE[0].value,
+      mode2: LAYOUT_MODE[1].value,
     },
   }
 
   state = {
-    galleryLayoutMode: this.props.hiddenFacets.layoutMode1 || LAYOUT_MODE[0].value,
+    mobileLayoutMode: this.props.mobileLayout.mode1,
```

Comment:
Please, double check this change, it was not working properly only with the props before.

Reply:
`this.props.mobileLayout.mode1` is already in the `defaultProps`.
But I also tested it out and it is working properly.